app.go

```diff
@@ -1000,3 +1053,76 @@ func (app *App) Fields() (map[string]*FieldInfo, error) {
 	}
 	return ret, nil
 }
+
+func (app *App) createCursor(fields []string, query string, size uint64) ([]byte, error) {
+	type cursor struct {
+		App    uint64   `json:"app"`
+		Fields []string `json:"fields"`
+		Size   uint64   `json:"size"`
+		Query  string   `json:"query"`
+	}
+	data := cursor{App: app.AppId, Fields: fields, Size: size, Query: query}
+	jsonData, _ := json.Marshal(data)
+	url := app.createUrl("records/cursor", "")
+	request, err := app.NewRequest("POST", url.String(), bytes.NewBuffer(jsonData))
+	if err != nil {
+		return nil, err
+	}
+	response, err := app.do(request)
+
+	if err != nil {
+		return nil, err
+	}
+	body, err := parseResponse(response)
+	if err != nil {
+		return nil, err
+	}
+	return body, nil
+}
+
+func (app *App) deleteCursor(id string) (string, error) {
+	type requestBody struct {
+		Id string `json:"id"`
+	}
+	data, err := json.Marshal(requestBody{Id: id})
+	if err != nil {
+		return "", err
+	}
+	url := app.createUrl("records/cursor", "")
+	request, err := app.NewRequest("DELETE", url.String(), bytes.NewBuffer(data))
+	if err != nil {
+		return "", err
+	}
+	response, err := app.do(request)
+	if err != nil {
+		return "", err
+	}
+
+	result, err := parseResponse(response)
+	if err != nil {
+		return "", err
+	}
+	if result != nil {
+	}
+	return "delete success", nil
```

Comment:
Please remove the returned string, this function only return if error occurred. Please refer to DeleteRecords function.